tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -174,19 +198,26 @@ def test_cpu_template(test_microvm_with_ssh, network_config, cpu_template):
     assert test_microvm.api_session.is_status_no_content(
             response.status_code)
 
+    # Check that all common features discoverable with lscpu
+    # are properly masked.
     ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
     guest_cmd = "cat /proc/cpuinfo | grep 'flags' | head -1"
     _, stdout, stderr = ssh_connection.execute_command(guest_cmd)
     assert stderr.read() == ''
 
-    cpu_flags_output = stdout.readline().rstrip()
+    cpu_flags_output = stdout.readline().rstrip().split(' ')
+
+    for feature in common_masked_features_lscpu:
+        assert feature not in cpu_flags_output, feature
+
+    # Check that all common features discoverable with cpuid
+    # are properly masked.
+    utils.check_guest_cpuid_output(test_microvm, "cpuid -1", None, '=',
```

Comment:
We run with T2 template by default?

Reply:
No we do not run with T2 by default. There are 3 scenarios:
* no templates are applied and there is no masked feature
* T2 template is run and the masked features are the ones defined in this test as `common_masked_features_cpuid` or `common_masked_features_lscpu`
* C3 template is run and all the masked features from the T2 template are included plus a set of additional features are masked and they are defined in the `c3_masked_features ` variable.
This test tests the last 2 scenarios, you can see that the test is parametrized with the `pytest.mark.parametrize("cpu_template", ["T2", "C3"])` label. So, it will run 2 times, once with the T2 template enabled and another time with the C3 enabled.
Is it clear now?

Reply:
Yep. Thanks. Resolving this.